Libraries/xcsdk/Headers/xcsdk/SDK/Manager.h

```diff
@@ -88,7 +89,7 @@ class Manager {
     /*
      * Load from a developer root. Returns nullptr on error.
      */
-    static std::shared_ptr<Manager> Open(libutil::Filesystem const *filesystem, std::string const &path);
+    static std::shared_ptr<Manager> Open(config::Config const *config, libutil::Filesystem const *filesystem, std::string const &path);
```

Comment:
nit: I'd put the `config` parameter last, after the `path`, since it's like additions to the path.